kits/daisy/README.md

```diff
@@ -0,0 +1,83 @@
+# Daisy Demo Code
+
+This folder contains everything needed to run the Daisy hexapod kit.  It assumes that you have:
+* Daisy Robot Kit (fully assembled)
+* iOS or Android mobile device (phone or tablet)
+
+## Unpacking Daisy
+The robot ships folded and packed in a single Pelican Case.  
+
+Included in the case is:
+- Daisy Robot
+- Gas Springs (6x)
+- Batteries (4x)
+- Chargers (2x)
+- WiFi Antennas (2x)
+
+The gas springs and WiFi antennas will need to be attached to the robot after unpacking.  
+
+## Starting Daisy
+
+#### Powering up the robot:
+- Mount and plug in at least 1 of the batteries (preferably both).  
+- [Turn on the batteries](https://www.ebikes.ca/product-info/ligo-batteries.html) by pressing and holding the button on the batteries until the lights flash green.
+- After the batteries turn on, you will hear 2 beeps, followed by 2 double-beeps as the router in the robot powers up.
+- Press and hold the button labeled 'Modules'.  When the power to the modules turns on the button will turn blue.
+- Turn on the power to the onboard computer.  When the power to the computer turns on the button will turn white.  *The computer does not need to be on for the modules in the robot to be seen on the network.*
+- Make sure the M-Stop button is off (not pressed in).
+
+#### Getting the robot ready to run:
+- **Make sure all of the legs are within their joint limits.** The [status LEDs](http://docs.hebi.us/core_concepts.html#led-status-codes) on the actuators will blink orange if the module is outside the joint limit.  When the actuators rotate to within their joint limits, the LEDs will change to a slow green fade.  The joints that will have to be rotated the most will be the ankle joints.  
+- You can also use [Scope](http://docs.hebi.us/tools.html#scope-gui) to check the position feedback and limits on each joint.
+- You can connect to the modules on the robot directly from an external computer by accessing the robot's WiFi network (default password `hebi1234`) or by plugging into the ethernet port on the back of the robot.  **If you are controlling the robot from an external computer, you should always use the wired ethernet connection.**
+
+#### Connecting a Mobile Device for Control:
+- The code to control the robot assumes that a mobile device is on the network running the [HEBI Mobile I/O App](http://docs.hebi.us/tools.html#mobile-io).  The device will need to be on the same network as the robot (usually connected to the robot's WiFi network), and it will need to be set to the Family `Daisy` and the Name `_Controller`.  You can set the Family/Name of the mobile device in [Scope](http://docs.hebi.us/tools.html#scope-gui), or in the settings for the app on the mobile device.
```

Comment:
We should probably standardize naming of modules and apps in the kit example code for various APIs.  (E.g., Rosie's "_Wheel[123]" convention).  I like the name of the robot as you have for the family here, and I can update the C++ examples to match; I'd prefer the name of the app be just "Controller" or "Mobile IO", though.